dataset.py

```diff
@@ -333,3 +308,48 @@ def tensor2im(image_tensor, imtype=np.uint8):
 def save_image(image_numpy, image_path):
     image_pil = Image.fromarray(image_numpy)
     image_pil.save(image_path)
+
+if __name__ == '__main__':
+    """
+    Main method
+    """
+    trainX, trainY = readData(TRAIN_FILENAME) # Training data
+    testX, testY = readData(TEST_FILENAME) # Testing data
+
+    parser = argparse.ArgumentParser()
```

Comment:
Remember to remove arguments before deploying to production

Reply:
We may still need arguments, in order to test on smaller batch files in production. Will be necessary for testing!

Reply:
ok that make sense